src/Uno.UI/UI/Xaml/Controls/DatePicker/DatePickerPresenter.xaml

```diff
@@ -0,0 +1,59 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" 
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:xamarin="http:///umbrella/ui/xamarin"
+      xmlns:android="http://nventive.com/android"
+      xmlns:ios="http://uno.ui/ios"
+      xmlns:u="using:Umbrella.View.Controls"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      mc:Ignorable="d xamarin ios android">
+
+	<ios:Style x:Key="DefaultDatePickerFlyoutPresenterStyle"
```

Comment:
Where is this style taken from? It doesn't match the windows one.

Reply:
The appearance matches the iOS style guidelines (display the flyout at the bottom). It was a taken from one of the projects that implemented the flyout correctly.